tools/istio-iptables/pkg/capture/run.go

```diff
@@ -131,7 +131,8 @@ func (cfg *IptablesConfigurator) logConfig() {
 	b.WriteString(fmt.Sprintf("ISTIO_EXCLUDE_INTERFACES=%s\n", os.Getenv("ISTIO_EXCLUDE_INTERFACES")))
 	b.WriteString(fmt.Sprintf("ISTIO_SERVICE_CIDR=%s\n", os.Getenv("ISTIO_SERVICE_CIDR")))
 	b.WriteString(fmt.Sprintf("ISTIO_SERVICE_EXCLUDE_CIDR=%s\n", os.Getenv("ISTIO_SERVICE_EXCLUDE_CIDR")))
-	b.WriteString(fmt.Sprintf("ISTIO_META_DNS_CAPTURE=%s", os.Getenv("ISTIO_META_DNS_CAPTURE")))
+	b.WriteString(fmt.Sprintf("ISTIO_META_DNS_CAPTURE=%s\n", os.Getenv("ISTIO_META_DNS_CAPTURE")))
+	b.WriteString(fmt.Sprintf("ISTIO_META_INVALID_DROP=%s\n", os.Getenv("ISTIO_META_INVALID_DROP")))
```

Comment:
maybe better to remove ISTIO_META? so we do not need to send it to pilot

Reply:
@hzxuzhonghu  What's the downside to add ISTIO_META prefix? It seems it's used to pass env vars as node metadata so we can get it in the pod environment so it will be straightforward to check in the sidecar if this flag is enabled or not, IMHO that's a good thing.
```
istio-proxy@swissknife-test-1:/$ echo $ISTIO_META_INVALID_DROP
true
```

Reply:
I mean we donot want to send this value via node metadata to pilot

Reply:
Ack, let me have another PR to remove the ISTIO_META prefix  @hzxuzhonghu

Reply:
Removed the prefix in PR https://github.com/istio/istio/pull/36667